Documentation/Changelog.md

```diff
@@ -1,8 +1,25 @@
 ## Changelog
 
+### 5.2.0
+
+This version wraps HockeySDK-iOS 5.1.2. It contains the following changes
+
+- [IMPROVEMENT] This release can be compiled with Xcode 9.2 without warnings. [#502](https://github.com/bitstadium/HockeySDK-iOS/pull/503)
+- [BUGFIX] Fix warnings when integrating the SDK as source in Xcode 9. [#501](https://github.com/bitstadium/HockeySDK-iOS/pull/501)
+- [BUGFIX] Fix a potential memory leak in `BITChannel`. [#500](https://github.com/bitstadium/HockeySDK-iOS/pull/500)
+- [BUGFIX] Version 5.1.X broke support for app extension. We're sorry about this and we've updated our test matrix to make sure this does not happen again. [#499](https://github.com/bitstadium/HockeySDK-iOS/pull/499)
+- [BUGFIX] Fix a bug in the Feedback UI when Feedback was shown in landscape. [#498](https://github.com/bitstadium/HockeySDK-iOS/pull/498)
+
```

Comment:
How about track event api?

Reply:
Good catch!

Reply:
This has been added already, no idea why Github doesn't display this correctly.